HousingRepairsOnlineApi/Domain/AppointmentTime.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using HACT.Dtos;
+
+namespace HousingRepairsOnlineApi.Domain
+{
+    public class AppointmentTime
+    {
+        public Reference Id { get; set; }
```

Comment:
Using `HACT.Dtos.Reference` type won't yield a single string Id - it has 3 string fields of `Id`, `Description` and `AllocatedBy`. It would also expose and rely upon the HACT data standard.

It would be better to use either a string or [GUID](https://docs.microsoft.com/en-us/dotnet/api/system.guid).